RawSpeed/DngDecoderSlices.cpp

```diff
@@ -143,6 +143,159 @@ my_error_throw (j_common_ptr cinfo)
   ThrowRDE("JPEG decoder error!");
 } 
 
+static void decodeFPDeltaRow(unsigned char * src, unsigned char * dst, size_t tileWidth, size_t realTileWidth, unsigned int bytesps, int factor) {
```

Comment:
Yes, it can be relicensed under LGPL, explanation is here:
http://softwareengineering.stackexchange.com/questions/105912/can-you-change-code-distributed-under-the-mit-license-and-re-distribute-it-unde